addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/internal/radiora/handler/RS232Handler.java

```diff
@@ -0,0 +1,130 @@
+
+/**
+ * The {@link RS232Handler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Jeff Lauterbach - Initial contribution
+ */
+public class RS232Handler extends BaseBridgeHandler implements RadioRAFeedbackListener {
+
+    private Logger logger = LoggerFactory.getLogger(RS232Handler.class);
+
+    private RadioRAConnection chronosConnection;
+
+    private RS232Config config;
+    private ScheduledFuture<?> zoneMapScheduledTask;
+
+    RS232Handler(Bridge bridge, RadioRAConnection connection) {
+        super(bridge);
+
+        this.chronosConnection = connection;
+    }
+
+    public RS232Handler(Bridge bridge) {
+        super(bridge);
+
+        this.chronosConnection = new RS232Connection();
+        this.chronosConnection.setListener(this);
+
+    }
+
+    @Override
+    public void dispose() {
+        if (zoneMapScheduledTask != null) {
+            zoneMapScheduledTask.cancel(true);
+        }
+
+        if (chronosConnection != null) {
+            chronosConnection.disconnect();
+        }
+    }
+
+    @Override
+    public void initialize() {
+        connectToChronos();
+
+        scheduleZoneMapQuery();
+    }
+
+    protected void connectToChronos() {
+        this.config = getConfigAs(RS232Config.class);
+        String portName = config.getPortName();
+        int baud = config.getBaud();
+
+        logger.debug("Attempting to connect to Chronos on port {}", portName);
+
+        if (!chronosConnection.open(portName, baud)) {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.HANDLER_INITIALIZING_ERROR,
+                    "Error initializing - Failed to connect to chronos");
+            return;
+        }
+
+        logger.debug("Connected successfully");
+
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    protected void scheduleZoneMapQuery() {
+        logger.debug("Scheduling zone map query at {} second inverval", config.getZoneMapQueryInterval());
+
+        zoneMapScheduledTask = this.scheduler.scheduleAtFixedRate(new Runnable() {
+
+            @Override
+            public void run() {
+                sendCommand(new ZoneMapInquiryCommand());
+            }
+
+        }, 3, config.getZoneMapQueryInterval(), TimeUnit.SECONDS);
+    }
+
+    public void sendCommand(RadioRACommand command) {
+        chronosConnection.write(command.toString());
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        logger.warn("Method handleCommand(...) not implemented");
+    }
+
+    @Override
+    public void handleRadioRAFeedback(RadioRAFeedback feedback) {
+        for (Thing thing : getThing().getThings()) {
+            ThingHandler handler = thing.getHandler();
+            if (handler instanceof LutronHandler) {
+                ((LutronHandler) handler).handleFeedback(feedback);
+            } else {
+                logger.error("Unexpected - Thing {} is not a LutronHandler", thing.getClass());
```

Comment:
In [E.4 of the coding guidelines](http://docs.openhab.org/developers/development/guidelines.html#e-logging) we listed what kind of logging level should be used when. Error should only be used:

> `error` logging should only be used to inform the user that something is tremendously wrong in his setup, the system cannot function normally anymore, and there is a need for immediate action. 

Please change this to use the correct logging levels and if possible/needed change the ONLINE/OFFLINE status of thing / bridge.

Can you check all?